pkg/volume/BUILD

```diff
@@ -54,6 +55,7 @@ go_library(
 go_test(
     name = "go_default_test",
     srcs = [
+        "gke_patch_test.go",
```

Comment:
Remove this

Reply:
done.

Jonathon suggested a different test to replace it but I'm not clear on what we're shooting for here.